File: DoAnLau-API/DoAnLau-API/DoAnLau-API/Models/Menu.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace DoAnLau_API.Models
{
    public class Menu
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
     
        public string menu_Id { get; set; }
        [Column( TypeName = "nvarchar(200)")]
        public string menuName { get; set; }
        [Column(TypeName = "varchar(400)")]
        public string menuImage { get; set; }
        public int price { get; set; }
        [Column(TypeName = "bit")]
        public bool hotDeal { get; set; }
        [Column(TypeName = "bit")]
        public bool bestSaller { get; set; }
        [Column(TypeName = "bit")]
        public bool state { get; set; }

        public MenuCategory menuCategory { get; set; }

        public ICollection<Order_Detail> order_Details { get; set; }

    }
}

File: file-angular/src/app/Models/Menu.ts
export class Menu {
  menu_Id: string;
  menuName: string;
  menuImage: string;
  price: number;
  hotDeal: boolean;
  bestSaller: boolean;
  state: boolean;

  constructor(
    menu_Id: string,
    menuName: string,
    menuImage: string,
    price: number,
    hotDeal: boolean,
    bestSaller: boolean,
    state: boolean
  ) {
    this.menu_Id = menu_Id;
    this.menuName = menuName;
    this.menuImage = menuImage;
    this.price = price;
    this.hotDeal = hotDeal;
    this.bestSaller = bestSaller;
    this.state = state;
  }
}
